server/sttp-mock-server/src/test/scala/sttp/tapir/server/mockserver/MockServerSpec.scala

```diff
@@ -537,6 +544,252 @@ class MockServerSpec extends AnyFlatSpec with Matchers {
     )
   }
 
+  it should "match json body without drop null" in {
```

Comment:
these tests are enormous. Is there a way to somehow deduplicate them, so that it would be visible what's the difference in the null handling, and where the nulls are in the first palce (or not)?

Reply:
OK, I'm on it